src/cadenza.js

```diff
@@ -591,7 +602,8 @@ function validMediaType(value) {
   const CSV = 'text/csv';
   const MS_EXCEL_2007 =
     'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet';
-  return value === CSV || value === MS_EXCEL_2007;
+  const PDF = 'application/pdf';
+  return !!value && [CSV, MS_EXCEL_2007, PDF].includes(value);
```

Comment:
For fetch/downloadData the mediaType is required, i.e. !!value is not correct.

Reply:
That was to fix some type errors, but you're right, I better fix the types in the JSDoc instead.